src/System.Text.Encoding/tests/ASCIIEncoding/ASCIIEncodingGetMaxByteCount.cs

```diff
@@ -2,27 +2,19 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
-using System.Collections.Generic;
 using Xunit;
 
 namespace System.Text.Tests
 {
     public class ASCIIEncodingGetMaxByteCount
     {
-        private static readonly RandomDataGenerator s_randomDataGenerator = new RandomDataGenerator();
```

Comment:
in general I am seeing the areas that uses or process input data buffers (like streams, encoding...etc.), using random data in some of the tests will have some value.

For this specific PR, I am ok with the change.